package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "ci": "ni && nr typecheck && nr lint && nr build && nr test:all",
+    "ci:docs": "run-s build docs:build",
```

Comment:
I don't know how many times we have to remove this  `npm-run-all` package 😄 

pnpm supports regexp syntax: `pnpm run /^(docs|docs:build)$/`

Reply:
The matched commands seem to be executed simultaneously, instead of executing `docs:build` after executing `build`. This is not as expected. 🤔

Reply:
I think `pnpm run --sequential /^(build|docs:build)$/` would do it, but since it's sequential, we could just chain with `&&`.

Reply:
Well, that works.